pytest_fixtures/api_fixtures.py

```diff
@@ -56,6 +56,11 @@ def default_location():
     return entities.Location().search(query={'search': f'name="{DEFAULT_LOC}"'})[0]
 
 
+@pytest.fixture(scope='function')
```

Comment:
explicit better than implicit?

Reply:
maybe I should add a comment `# ITS FUNCTION SCOPED BRO`